README.md

```diff
@@ -43,6 +43,10 @@ See https://github.com/clearcontainers/runtime/wiki/Installation.
 
 See https://github.com/hyperhq/runv.
 
+# Governance
+
+The Kata Containers project is governed according to the [“four opens"](https://governance.openstack.org/tc/reference/opens.html), which are open source, open design, open development and open community. Technical decisions will be made by technical contributors and a representative Architecture Committee. The community is committed to diversity, openness, encouraging new contributors and leaders to rise up.
+
 ## Developers
 
```

Comment:
semi-unrelated nit: I thought I'd (asked to have) changed:

` who has merged code within the last 12 months`
to
`who has had code merged within the last 12 months`

Or was that in another doc/repo?

Reply:
Good catch - fixing.. Not clear on your question though I'm afraid?

Reply:
I thought I'd caught this before - maybe it was the same phrase in another doc though - np - thanks for fixing!